modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/related/models/test/ObjectRelatedModelsProviderTest.java

```diff
@@ -437,6 +437,14 @@ public void testObjectEntry1toMObjectRelatedModelsProviderImpl()
 			_objectRelationship.getObjectRelationshipId(),
 			objectEntry8.getObjectEntryId());
 
+		// Remove object entry to assert empty one to many relationship
```

Comment:
@G-Sa I did not understand the `to assert empty one to many relationship`. I did not see any `assert` here.